interface/patient_file/summary/demographics_save.php

```diff
@@ -30,6 +30,10 @@
   "WHERE form_id = 'DEM' AND uor > 0 AND field_id != '' " .
   "ORDER BY group_name, seq");
 while ($frow = sqlFetchArray($fres)) {
+  //continue if field is defined as disabled in the LBF (disabled field in the html form doesn't sent by submit but it isn't empty field)
+  if(strpos($frow['edit_options'], '0') !== false){
+      continue;
+  }
```

Comment:
I think your work week starts again tomorrow. Agree with Rod that seems best to instead use (and also note this will be an issue that likely needs fixing in the other layout save scripts):
``````
isset($_POST["form_$field_id"])